node/hack/governor/src/index.ts

```diff
@@ -224,7 +224,9 @@ axios
 
     for (var token of existingTokenKeys) {
       // A token has been removed from the token list 
-      if (!newTokenKeys[token]) {
+      // We cut the symbol off the end of the key as it's possible for a token to change its symbol
+      var tokenParts = token.split("-");
+      if (!newTokenKeys[tokenParts[0] + "-" + tokenParts[1]]) {
```

Comment:
Any risk here that `tokenParts` will be empty or that its length can be < 2? If so you might get a NaN or null or crash or "oopsie-doopsie" or however JS/TS handles these things

Reply:
Every `token` is read from the previously generated token list, so no. If the token list were malformed then it would have broken on the previous guardian update anyway and we have tests to check the validity of the generated token list

Reply:
lol, fwiw, you get `undefined` which would then get coerced so you would then get `undefined-undefined` on an empty list.